packages/story-editor/src/components/library/panes/media/common/innerElement.js

```diff
@@ -201,7 +201,7 @@ function InnerElement({
             />
           )}
         </Video>
-        {!newVideoPosterRef.current && (
+        {!newVideoPosterRef.current && poster && (
```

Comment:
Couldn't we use the `displayPoster` const here?

Reply:
Yeah it could be used.